libs/api/domains/national-registry/src/lib/types/user.type.ts

```diff
@@ -31,4 +31,7 @@ export interface User {
     city: string
     date: string
   }
+  spouseName?: string
+  spouseNationalId?: string
+  spouseCohab?: string
```

Comment:
Lets have a spouse object that's optional, then we wont have to check each property if no spouse
```
spouse?: {
  name?: string,
  nationalId?: string,
  cohab?: boolen, // boolean not string, right?
}
```

Reply:
Cohab is actually a string, it can return all sorts of different results 😃